### PR TITLE
Created Folders

### PR DESCRIPTION
**Pull request by Colividiu**
I added the folders that are commonly found in React apps, complete with short descriptions of what each one will contain

### CODE REVIEW

File: src/actions/reducers/reducers-info [hunk 1]
```diff
@@ -0,0 +1 @@
+Redux-specific code goes here (In case the ned arises)
```
Comment (tomoiagahoratiu) [inline src/actions/reducers/reducers-info]: we will not need reducers for now, you can delete this dir

File: src/actions/components/components-info [hunk 2]
```diff
@@ -0,0 +1 @@
+React components go here
```
Comment (tomoiagahoratiu) [inline src/actions/components/components-info]: I would also delete the actions directory and put these directories inside src directly

Comment (Colividiu) [reply src/actions/components/components-info]: Got it. I'm on it

**Comment by Colividiu** [comment]
There, it should all be changed according to your suggestions now!

**Comment by tomoiagahoratiu** [comment]
Good, now because you merged the other PR, please do a rebase with main and then use git push -f. The rebase we do in order to keep our branch up to date with the main branch and see if there are any conflicts when merging.

**Comment by tomoiagahoratiu** [comment]
A good practice is to always rebase your PR branch before doing the merge

**Review by tomoiagahoratiu — APPROVED** [review]

**Comment by Colividiu** [comment]
Is it ok now? (making sure)

**Comment by tomoiagahoratiu** [comment]
Yes, you can merge

**Comment by Colividiu** [comment]
Also, if you have the time, why is -f used here? I was aware that it is considered a bad practice to use force in general when working on github. I am assuming that this is however acceptable since I am not overwriting someone else's history.

**Comment by tomoiagahoratiu** [comment]
Taking the following example:
Main branch git history:
-commit1
-commit2

your branch when creating it from main:
-commit1
-commit2

We do some changes to our branch and do a commit, the history changes to:
-changeCommitBranchCreatedFromMain
-commit1
-commit2

Between the time that we want to merge someone does a merg in main and the main history becomes:
-changeToMainBranch
-commit1
-commit2

So the status of our branch is behind with a commit from the main branch (changeToMainBranch) but with one ahead (changeCommitBranchCreatedFromMain). For merging our branch to main we need to be up to date and have all commits from main to our branch, but we already pushed one commit. So in order to keep the same history to the commits we do a rebase with main which introduce the missing commits behind of our commits. Our branch history will become:
From this:
-changeCommitBranchCreatedFromMain
-commit1
-commit2

To this:
-changeCommitBranchCreatedFromMain(our changes)
-changeToMainBranch (insertion of all missing commits from main to our branch)
-commit1
-commit2

As displayed above we are changing the history of our branch so we need to do a push with force. If this is not clear please follow the resource below. If you still have questions I can probably show you when we have a live session.
All of this is displayed here in this game with a better of visualisation of how the git works https://learngitbranching.js.org/

**Comment by Colividiu** [comment]
That is... a really great explanation! Thanks a lot, things started clearing up in my mind instantly!

**Pull request merged**